src/components/MarketList/index.js

```diff
@@ -94,7 +94,10 @@ class MarketList extends Component {
           {isOwner &&
             !isResolved && (
               <div className="market__control">
-                <a href="javascript:void(0)" onClick={e => this.handleViewMarketResolve(e, `/markets/${market.address}/resolve`)}>
+                <a
```

Comment:
This method should be placed in a separate constant otherwise always this component will be re-rendered even if it not necessary.